src/server/pps/pretty/pretty.go

```diff
@@ -47,22 +47,14 @@ func PrintJobInfo(w io.Writer, jobInfo *ppsclient.JobInfo) {
 func PrintPipelineHeader(w io.Writer) {
 	// because STATE is a colorful field it has to be at the end of the line,
 	// otherwise the terminal escape characters will trip up the tabwriter
-	fmt.Fprint(w, "NAME\tINPUT\tOUTPUT\tSTATE\t\n")
```

Comment:
What about something like `Repo1 X (Repo2 U Repo3)` it seems like that would be a pretty readable shorthand.